src/mood.ts

```diff
@@ -136,9 +154,17 @@ class SkillMoodElement extends MoodElement {
   execute(mood: Mood, ensureTurns: number): boolean {
     const effect = toEffect(this.skill);
     const initialTurns = haveEffect(effect);
+    let showerShieldUnequipped = false;
```

Comment:
instead of storing this in a `let` variable, use a `const` to store the slot it's equipped in:
```typescript
  const aprilShowerShieldSlot = Slot.all().find((s) => equippedItem(s) === $item`April Shower Thoughts shield`);
```
```
if (this.skill === $skill`Empathy of the Newt` && aprilShowerShieldSlot) { 
  unequip(aprilShowerShieldSlot);
}
```
```
if (aprilShowerShieldSlot) equip(aprilShowerShieldSlot, $item`April Shower Thoughts shield`);
```